docs/catalog.py

```diff
@@ -44,10 +69,23 @@ def make_content(artifact, label, all_labels):
         result += "+" * len(explanation_str) + "\n\n"
         result += artifact_class.__doc__ + "\n"
 
+    subtypes = all_subtypes_of_artifact(artifact)
+    subtypes = list(set(subtypes))
+    subtypes.remove(artifact_type)  # this was already documented
+    for subtype in subtypes:
+        subtype_class = Artifact._class_register.get(subtype)
+        subtype_class_name = subtype_class.__name__
+        if subtype_class.__doc__:
+            explanation_str = f"Explanation about `{subtype_class_name}`"
```

Comment:
suggestion: i think this is based on another place where an explanation is added to the rst text? if so, and its the same code in the other place, maybe consider making it a small function ..

Reply:
This is what i am referring to, just above: https://github.com/IBM/unitxt/pull/528/files#diff-380b97fbfb3b8253f06f95f85d25f212caa0ef7b146e079429c8a5a8057f70fcL42